src/test/scala/examples/examples.scala

```diff
@@ -0,0 +1,59 @@
+package examples
+
+import akka.actor.Actor.Receive
+import akka.actor.{Props, ActorRef, Actor, ActorSystem}
+import akka.stream.ActorMaterializer
+import akka.stream.actor.WatermarkRequestStrategy
+import com.softwaremill.react.kafka.{ConsumerProps, ProducerProps, ReactiveKafka}
+import kafka.serializer.{StringDecoder, StringEncoder}
+
+/**
+ * Code samples for the documentation.
+ */
+object examples {
+
+  def simple(): Unit = {
+
+    import com.softwaremill.react.kafka.ReactiveKafka
+    import com.softwaremill.react.kafka.ProducerProps
+    import com.softwaremill.react.kafka.ConsumerProps
+    import akka.actor.ActorSystem
+    import akka.stream.ActorMaterializer
+    import akka.stream.scaladsl.{Sink, Source}
```

Comment:
move imports to begin of file? (or if you prefer make `handling` consistent with respect to place of imports)